modules/telemetry/src/specific/planetssonification.cpp

```diff
@@ -0,0 +1,779 @@
+    _marsProperty.temperatureEnabled.onChange([this]() { onMarsSettingChanged(); });
+    _marsProperty.atmosphereEnabled.onChange([this]() { onMarsSettingChanged(); });
+    _marsProperty.moonsEnabled.onChange([this]() { onMarsSettingChanged(); });
+    // No rings
+    addPropertySubOwner(_marsProperty);
+
+    // Jupiter
+    _jupiterProperty.toggleAll.onChange([this]() { onJupiterAllChanged(); });
+    _jupiterProperty.sizeDayEnabled.onChange([this]() { onJupiterSettingChanged(); });
+    _jupiterProperty.gravityEnabled.onChange([this]() { onJupiterSettingChanged(); });
+    _jupiterProperty.temperatureEnabled.onChange([this]() { onJupiterSettingChanged(); });
+    _jupiterProperty.atmosphereEnabled.onChange([this]() { onJupiterSettingChanged(); });
+    _jupiterProperty.moonsEnabled.onChange([this]() { onJupiterSettingChanged(); });
+    // No rings visible in OpenSpace
+    addPropertySubOwner(_jupiterProperty);
+
+    // Saturn
+    _saturnProperty.toggleAll.onChange([this]() { onSaturnAllChanged(); });
+    _saturnProperty.sizeDayEnabled.onChange([this]() { onSaturnSettingChanged(); });
+    _saturnProperty.gravityEnabled.onChange([this]() { onSaturnSettingChanged(); });
+    _saturnProperty.temperatureEnabled.onChange([this]() { onSaturnSettingChanged(); });
+    _saturnProperty.atmosphereEnabled.onChange([this]() { onSaturnSettingChanged(); });
+    _saturnProperty.moonsEnabled.onChange([this]() { onSaturnSettingChanged(); });
+    _saturnProperty.ringsEnabled.onChange([this]() { onSaturnSettingChanged(); });
+    addPropertySubOwner(_saturnProperty);
+
+    // Uranus
+    _uranusProperty.toggleAll.onChange([this]() { onUranusAllChanged(); });
+    _uranusProperty.sizeDayEnabled.onChange([this]() { onUranusSettingChanged(); });
+    _uranusProperty.gravityEnabled.onChange([this]() { onUranusSettingChanged(); });
+    _uranusProperty.temperatureEnabled.onChange([this]() { onUranusSettingChanged(); });
+    _uranusProperty.atmosphereEnabled.onChange([this]() { onUranusSettingChanged(); });
+    _uranusProperty.moonsEnabled.onChange([this]() { onUranusSettingChanged(); });
+    // No rings visible in OpenSpace
+    addPropertySubOwner(_uranusProperty);
+
+    // Neptune
+    _neptuneProperty.toggleAll.onChange([this]() { onNeptuneAllChanged(); });
+    _neptuneProperty.sizeDayEnabled.onChange([this]() { onNeptuneSettingChanged(); });
+    _neptuneProperty.gravityEnabled.onChange([this]() { onNeptuneSettingChanged(); });
+    _neptuneProperty.temperatureEnabled.onChange([this]() { onNeptuneSettingChanged(); });
+    _neptuneProperty.atmosphereEnabled.onChange([this]() { onNeptuneSettingChanged(); });
+    _neptuneProperty.moonsEnabled.onChange([this]() { onNeptuneSettingChanged(); });
+    // No rings visible in OpenSpace
+    addPropertySubOwner(_neptuneProperty);
+
+    // Precision
+    _anglePrecision = _precisionProperty.lowAnglePrecision;
+    _distancePrecision = _precisionProperty.lowDistancePrecision;
```

Comment:
```suggestion
    _anglePrecision = _precisionProperties.lowAnglePrecision;
    _distancePrecision = _precisionProperties.lowDistancePrecision;
```
See previous comment